docs/content/users/extend/custom-commands.md

```diff
@@ -73,6 +73,7 @@ A number of environment variables are provided to these command scripts. These a
 * `DDEV_HOST_HTTPS_PORT`: localhost port for HTTPS on web server
 * `DDEV_HOST_MAILHOG_PORT`: localhost port for MailHog
 * `DDEV_HOST_WEBSERVER_PORT`: localhost port of the web server
+* `DDEV_MUTAGEN_ENABLED`: "true" if mutagen is enabled
```

Comment:
```suggestion
* `DDEV_MUTAGEN_ENABLED`: `true` if Mutagen is enabled
```

Reply:
The oddity about this one, and the reason I had left it with quotes, is the difference between the text string "true" and the binary `true`. However, there isn't a difference in the context of an environment variable, so going with your suggestion. It makes enormous difference in yaml.

Reply:
In that case it could just as well be `"true"` to express that I think. I had looked at `IS_DDEV_PROJECT` on this page and assumed this should follow suit.